src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
The white points listed for Srgb are using the D65 space. So the white point needs to be hard coded to use D65. This is the reason why I was suggesting the RgbStandard have a separate white point. Then we can use the chromatic adaptation to automatically convert. Not sure if there is a better way to implement this.

Here is a link that shows the transformation from Srgb to ICC Srgb D50
http://ninedegreesbelow.com/photography/srgb-color-space-to-profile.html#sRGB-profile-source-destination-white-points